src/Autofac/Core/Activators/Reflection/ReflectionActivator.cs

```diff
@@ -78,6 +83,40 @@ public void ConfigurePipeline(IComponentRegistryServices componentRegistryServic
             throw new ArgumentNullException(nameof(pipelineBuilder));
         }
 
+#if NET7_0_OR_GREATER
+        // The 'inherited' flag on GetCustomAttribute does not appear to work as expected with regards to RequiredMemberAttribute,
```

Comment:
WAT

Is there an issue filed for this somewhere?

Reply:
Oh, that would make sense.

Reply:
I'll update the comment to say that.